requirements/prod.txt

```diff
@@ -1,9 +1,9 @@
 ## production requirements
 
-Django>=1.11.15,<2.0
+Django>=2.2,<3.0
 pillow>=5.2.0
-wagtail<=2.3
-mezzanine<=4.3
```

Comment:
Please add a comment documenting briefly what "version" of mezzanine this is (or something). Maybe say Django 2.2 compatible version as of x date ?